crates/krun-server/src/cli_options.rs

```diff
@@ -0,0 +1,38 @@
+use bpaf::{any, construct, env, positional, OptionParser, Parser};
+
+#[derive(Clone, Debug)]
+pub struct Options {
+    pub server_port: u32,
+    pub command: String,
+    pub command_args: Vec<String>,
+}
+
+pub fn options() -> OptionParser<Options> {
+    let server_port = env("KRUN_SERVER_PORT")
+        .help("TCP port to listen for command launch requests")
+        .argument::<u32>("KRUN_SERVER_PORT")
+        .fallback(3333);
```

Comment:
```suggestion
    let port = long("port")
        .short('p')
        .env("KRUN_SERVER_PORT")
        .help("TCP port to listen for command launch requests")
        .argument("PORT")
        .fallback(3333)
        .display_fallback();
```

Reply:
Uhh... Sorry, I see now that it's quite pointless to have long / short options here.